### PR TITLE
Length validations

### LINKED ISSUE #215: Long field values break Redshift
Redshift's `character varying` only allows 256 characters. But Postgres's default for that type is 1GB. After batting this back and forth, Jon and I decided to do two things:

1. Apply sane length validations on fields. 
* Don't send Redshift values with lengths that will break the sync.

### PR DESCRIPTION
**Pull request by jonallured**
Addressing this from two angles:

* ensure that varchar-type fields have sensible limits on their length
* ensure that lists of things (tag and source names) don't exceed limits at export time

closes #215

### CODE REVIEW

File: app/models/organization_resolver.rb [hunk 1]
```diff
@@ -26,7 +28,7 @@ def find_highest_rankable(rankables)
     rankables.order(created_at: :desc).sort_by(&:rank).first
   end
 
-  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```
Comment (gnilekaw) [inline app/models/organization_resolver.rb]: :shaking_fist: rubbboooocop!

**Comment by gnilekaw** [comment]
These limits are based on Redshift values, but I'm gonna hold my peace ;)

**Comment by jonallured** [comment]
FWIW, I feel your pain @gnilekaw :(

My thought it we start with the validations and the truncations being equal. If we later decide that we want to accept longer inputs, we could change that on the model side and still have the export side limit what's being sent in the CSVs. But I wanted to do that in response to something and this seemed like a good way to get started.

**Comment by gnilekaw** [comment]
Sigh. I agree. The limits might be totally fine, in which case it's the right decision. Let's only revisit if it causes issues with real-world data.

**Pull request merged**